package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "increment",
+  "private": true,
+  "version": "0.0.0-development",
+  "license": "MIT License",
+  "repository": "https://github.com/step-security/increment",
+  "scripts": {
+    "lint": "eslint index.js",
+    "build": "ncc build index.js -m -o dist",
+    "build_stats": "ncc build index.js -s --stats-out dist/stats -o dist"
+  },
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^5.1.1",
+    "axios": "^1.7.7"
+  },
+  "devDependencies": {
+    "@vercel/ncc": "^0.38.2",
+    "eslint-config-strongloop": "^2.1.0",
+    "eslint-plugin-jsdoc": "^50.0.0",
+    "husky": "^9.0.7",
```

Comment:
lets remove husky and related dependencies

Reply:
done